backend/tarsy/repositories/history_repository.py

```
@@ -486,9 +486,9 @@ def get_alert_sessions(
                 filters_applied={}
             )
 
-    def get_session_timeline(self, session_id: str) -> Optional[DetailedSession]:
+    def get_session_details(self, session_id: str) -> Optional[DetailedSession]:
         """
-        Reconstruct chronological timeline for a session.
+        Get complete session details including chronological timeline, stages, and all interactions.
```

Ruff (0.12.2)

Line too long (100 > 88)

(E501)